src/eval-frame/components/eval-container.jsx

```diff
@@ -14,10 +15,24 @@ export class EvalContainerUnconnected extends React.Component {
   static propTypes = {
     reportOnly: PropTypes.bool.isRequired,
   }
+  state = {
+    hasError: false,
+    error: null,
+    errorInfo: null,
+  }
+  componentDidCatch(error, info) {
+    this.setState({
+      hasError: true,
+      error,
+      errorInfo: info,
+    })
+    console.log(error, info)
```

Comment:
We should remove this `console.log` before landing.

Reply:
Yes I do this is just for checking.

Reply:
you need to actually take out the line before you can mark this as resolved, thanks!